docs/reference/mapping/params/properties.asciidoc

```diff
@@ -58,10 +56,9 @@ PUT my_index/_doc/1 <4>
 }
 --------------------------------------------------
 // CONSOLE
-<1> Properties under the `_doc` mapping type.
```

Comment:
Is this explanation maybe still useful to keep, without mention of a 'mapping type' (something like "Properties in the top-level mappings definition.")?

Reply:
I changes this.